src/features/MoviesBrowser/ListPage/PersonTile/index.js

```diff
@@ -1,12 +1,23 @@
-import { Link } from "react-router-dom";
+
 import { toPerson } from "../../../../core/config/routes";
+import { StyledPersonTile, StyledLink, StyledPoster, PersonDetails } from "./styled";
+
+const PersonTile = ({ person, showJob, showCharacter }) => {
 
-const PersonTile = ({ person }) => {
     return (
-        <div>
-            <img alt={person.name} src={`https://image.tmdb.org/t/p/w200${person.profile_path}`} />
-            <Link to={toPerson(person)}>{person.name}</Link>
-        </div>
+        <StyledPersonTile>
+            <StyledLink
```

Comment:
tutaj zrobiłaś mutanta xD połączyłaś `<StyledPoster>` i `<StyledLink>` w jeden komponent :P

powinno być
```jsx
<StyledLink  to={toPerson(person)}>
<StyledPoster alt={person.name} src={`https://image.tmdb.org/t/p/w200${person.profile_path}`} />
{person.name}
</StyledLink>
```

Reply:
Widzisz. Za dużo Marvela :P A to jest Hulk :D